NumberBaseball/NumberBaseball/main.swift

```diff
@@ -2,9 +2,63 @@
 //  NumberBaseball - main.swift
 //  Created by yagom. 
```

Comment:
```suggestion
//  Created by 애종, LJ. 
```

이런 사소한 부분도 고쳐주시면 좋습니다